presto-main/src/main/java/com/facebook/presto/sql/planner/PlanBuilder.java

```diff
@@ -79,6 +82,11 @@ public Symbol translate(Expression expression)
         return translations.get(expression);
     }
 
+    public FunctionCall rewriteGroupingOperationToFunctionCall(GroupingOperation expression, QuerySpecification node)
```

Comment:
Yes, good suggestion. I had a quick look and I don't think there's a reason why we need to pollute either `PlanBuilder` or `TranslationMap`. I'll extract the method in a new commit.